pages/common/git-effort.md

```diff
@@ -0,0 +1,25 @@
+# git effort
+
+> Display how much activity a file has had, showing commits per file and active days.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-effort>.
+
+- Display each file, showing commits and active days:
+
+`git effort`
+
+- Display files receiving over a specific number of commits, showing commits and active days:
+
+`git effort --above {{5}}`
+
+- Display files committed to, from a specific author, showing commits and active days:
+
+`git effort -- --author"{{username}}"`
+
+- Display files committed to, from a specific time/date, showing commits and active days:
+
+`git effort -- --since='{{last month}}'`
+
+- Display files committed to, from specific chosen files, showing commits and active days:
```

Comment:
```suggestion
- Display each file in the specified directory, showing commits and active days:
```

Reply:
But it doesn't, if its directory/* it does

Reply:
Added new example in https://github.com/tldr-pages/tldr/pull/6002/commits/ec123302d3f4306f6a3aa2934158ca0ce4799fe4

Keep suggesting however once you had read this suggest a new one

Reply:
Ah, then maybe this?
```suggestion
- Display only the specified file or directory, showing commits and active days:
```